engine/src/main/java/org/hibernate/search/reader/impl/ManagedMultiReader.java

```diff
@@ -67,47 +70,42 @@ static ManagedMultiReader createInstance(IndexManager[] indexManagers, Iterable<
 			readerProviders[index] = indexReaderManager;
 		}
 
-		IndexReader[] effectiveReaders = getEffectiveReaders( indexManagers, subReaders, configuredSortableFields, sort );
+		IndexReader[] effectiveReaders = getEffectiveReaders( indexManagers, subReaders, configuredSorts, sort );
 		return new ManagedMultiReader( effectiveReaders, subReaders, readerProviders );
 	}
 
 	/**
-	 * Gets the readers to be effectively used. Will be the given readers, if:
+	 * Gets the readers to be effectively used. A given reader will be returned itself if:
```

Comment:
No, not really. Because it's not the reader which returns something but this method.